docs/src/assets/page-utils.js

```diff
@@ -43,6 +43,17 @@ export function copyHeading (id) {
   })
 }
 
+export function copyPropName (propName) {
+  copyToClipboard(propName)
+
+  Notify.create({
+    message: 'Prop name has been copied to clipboard.',
+    position: 'top',
+    actions: [{ icon: 'cancel', color: 'white', dense: true, round: true }],
+    timeout: 2000
+  })
+}
```

Comment:
`copyHeading` which is kind of similar to this helper, is used in 3 different places. But, this one is only being used in one place. So, I would put this into where it's used.